src/main/java/com/backend/webapp/controller/LoginController.java

```diff
@@ -37,7 +42,7 @@ public class LoginController {
     public ResponseEntity performLogin(@RequestBody LoginRequest loginRequest) {
         if (null == loginRequest.getEmail() || null == loginRequest.getPassword()) {
```

Comment:
```suggestion
        if (loginRequest.getEmail() == null || loginRequest.getPassword()  == null ) {
```
Computer science fundamentals xD

Reply:
-_-

Reply:
habit

Reply:
correct it lol

Reply:
???

Reply:
check model classes